pkg/cmd/server/origin/master.go

```diff
@@ -282,7 +282,6 @@ func (c *MasterConfig) buildHandlerChain(genericConfig *apiserver.Config) (func(
 
 			// these handlers are all before the normal kube chain
 			handler = serverhandlers.TranslateLegacyScopeImpersonation(handler)
-			handler = cacheControlFilter(handler, "no-store") // protected endpoints should not be cached
```

Comment:
@deads2k Was this the cause behind openapi data not being cached?

Reply:
> @deads2k Was this the cause behind openapi data not being cached?

yes